Makefile

```
@@ -423,7 +423,8 @@ deploy/integreatly-rhmi-cr.yml:
 	sed "s/OPERATORS_IN_PRODUCT_NAMESPACE/$(OPERATORS_IN_PRODUCT_NAMESPACE)/g" | \
 	sed "s/USE_CLUSTER_STORAGE/$(USE_CLUSTER_STORAGE)/g" > config/samples/integreatly-rhmi-cr.yml
 	# Workaround until in_prow annotation can be removed from prow
-	yq w -i config/samples/integreatly-rhmi-cr.yml metadata.annotations.in_prow "IN_PROW"
+	yq e -i '.metyqadata.annotations.in_prow="IN_PROW"' config/samples/integreatly-rhmi-cr.yml
```

looks like a typo here
```suggestion
	yq e -i '.metaadata.annotations.in_prow="IN_PROW"' config/samples/integreatly-rhmi-cr.yml
```

---

typo my own suggestion :facepalm:

---

Typo is fixed.